Addons/tsgExoticQuadrature.hpp

```diff
@@ -92,7 +100,11 @@ inline std::vector<std::vector<double>> getRoots(const int n, const std::vector<
     return roots;
 }
 
-//! \brief Evaluates a Lagrange basis polynomial at a point \b x.
+/*!
+ * \internal
+ * \brief Evaluates a Lagrange basis polynomial at a point \b x.
```

Comment:
Add the internal methods the Doxygen group. We'll improve the documentation later, but we can put things in the right spot now.

Reply:
Done.